contracts/MechMarketplace.sol

```diff
@@ -196,6 +206,29 @@ contract MechMarketplace is IErrorsMarketplace {
         );
     }
 
+    /// @dev Calculates payment for request delivery.
+    /// @param mech Delivery mech address.
+    /// @param payment Payment amount.
+    function _calculatePayment(
```

Comment:
Maybe we should move this to the side of a factory or a mech itself? Just make sure mech is not able to manipulate. This would allow to proceed with a default scenario.